### ISSUE TITLE
Should include index.html in the example

### ISSUE THREAD

**Issue by AnsonT** [d4rky-pl/arboris#24]
To be complete, it's helpful to include the index.html template with the rest of the example.

**Comment by d4rky-pl** [comment]
Good idea! The documentation could actually use a lot of work but we're
still kind of waiting to see in which way React is going to evolve (since
componentWillMount got deprecated but Suspense API is still not ready)
which means the project is temporarily on hold. It still works and it's
used in production but we'd rather not spend too much time polishing it as
a full rewrite is planned.


I will, however, keep this issue open so when we actually get to work we
remember about it when designing new docs. Thanks!
